docs/source/peers/peers.md

```diff
@@ -160,6 +167,22 @@ Later in this section, you'll learn more about the detailed nature of this
 ordering process --- and for a really detailed look at this process see the
 [Transaction Flow](../txflow.html) topic.
 
+### Using Fabric Gateway
+
+Hyperledger Fabric v2.4 introduces the optional Fabric Gateway to simplify client
+application development. Fabric Gateway (the gateway) migrates code for transaction
+proposals and endorsements to the gateway peer, which then interacts with the channels
+and peers required by the transaction. The legacy model of transaction proposal and
+endorsement code residing in the client application remains supported. For those
```

Comment:
Again, I wouldn't use the word "code" here. The endorsement management logic was in the SDK that the client application depends on, not in the client application itself.

Reply:
ok